lib/blocks.php

```diff
@@ -312,7 +312,7 @@ function strip_dynamic_blocks_add_filter( $text ) {
 	 * @return string
 	 */
 	function strip_dynamic_blocks_remove_filter( $text ) {
-		remove_filter( 'the_content', 'strip_dynamic_blocks', 8 );
+		remove_filter( 'the_content', 'strip_dynamic_blocks', 6 );
```

Comment:
Should we have a constant to share that `6` between the `_add_` and `_remove_` pair?